po/pt_BR.po

```diff
@@ -94,106 +99,142 @@ msgid "Other"
 msgstr "Outro"
 
 #: lib/logitech_receiver/notifications.py:67
-msgid "closed"
-msgstr "fechado"
+msgid "pairing lock is closed"
+msgstr "trava de emparelhamento está fechada"
 
 #: lib/logitech_receiver/notifications.py:67
-msgid "open"
-msgstr "aberto"
+msgid "pairing lock is open"
+msgstr "trava de emparelhamento está aberta"
 
-#: lib/logitech_receiver/notifications.py:67
-msgid "pairing lock is "
-msgstr "trava de emparelhamento está "
-
-#: lib/logitech_receiver/notifications.py:150 lib/solaar/ui/notify.py:118
+#: lib/logitech_receiver/notifications.py:150 lib/solaar/ui/notify.py:132
 msgid "unpaired"
 msgstr "desemparelhado"
 
-#: lib/logitech_receiver/notifications.py:192
+#: lib/logitech_receiver/notifications.py:193
 msgid "powered on"
 msgstr "ligado"
 
-#: lib/logitech_receiver/receiver.py:107 lib/solaar/ui/window.py:622
+#: lib/logitech_receiver/receiver.py:107 lib/solaar/ui/window.py:611
 msgid "unknown"
 msgstr "desconhecido"
 
-#: lib/logitech_receiver/settings_templates.py:77
+#: lib/logitech_receiver/settings_templates.py:119
 msgid "Smooth Scrolling"
 msgstr "Rolagem suave"
 
-#: lib/logitech_receiver/settings_templates.py:78
+#: lib/logitech_receiver/settings_templates.py:120
+#: lib/logitech_receiver/settings_templates.py:124
 msgid "High-sensitivity mode for vertical scroll with the wheel."
 msgstr "Modo de alta-sensibilidade para rolagem vertical com a roda."
 
-#: lib/logitech_receiver/settings_templates.py:79
+#: lib/logitech_receiver/settings_templates.py:121
+msgid "High Resolution Wheel Invert"
+msgstr "Roda Invertida de Alta Resolução"
+
+#: lib/logitech_receiver/settings_templates.py:122
+msgid "High-sensitivity wheel invert mode for vertical scroll."
+msgstr "Roda invertida de alta sensibilidade para rolagem vertical."
+
+#: lib/logitech_receiver/settings_templates.py:123
+msgid "Wheel Resolution"
+msgstr "Resolução da Roda"
+
+#: lib/logitech_receiver/settings_templates.py:125
 msgid "Side Scrolling"
 msgstr "Rolagem Lateral"
 
-#: lib/logitech_receiver/settings_templates.py:80
+#: lib/logitech_receiver/settings_templates.py:126
 msgid ""
 "When disabled, pushing the wheel sideways sends custom button events\n"
 "instead of the standard side-scrolling events."
 msgstr ""
-"Quando desabilitado, pressionar a roda lateralmente envia eventos customizados\n"
+"Quando desabilitado, pressionar a roda lateralmente envia eventos "
+"customizados\n"
 "ao invés dos eventos tradicionais de rolagem lateral."
 
-#: lib/logitech_receiver/settings_templates.py:82
+#: lib/logitech_receiver/settings_templates.py:128
 msgid "Sensitivity (DPI)"
 msgstr "Sensibilidade (DPI)"
 
-#: lib/logitech_receiver/settings_templates.py:83
+#: lib/logitech_receiver/settings_templates.py:129
+msgid "Sensitivity (Pointer Speed)"
+msgstr "Sensibilidade(Velocidade do Ponteiro)"
```

Comment:
Is it usual in portguese to have no space before `(`?

Reply:
Sorry, forgot to add an space there. Just pushed a fixed version.